src/amo/components/SecondaryHero/styles.scss

```diff
@@ -0,0 +1,63 @@
+@import '~amo/css/styles';
+
+.SecondaryHero {
+  color: $black;
+  display: flex;
+  justify-content: space-between;
+  min-height: 216px;
+  padding: $padding-page-l $padding-page;
+}
+
+.SecondaryHero-message {
+  font-size: 16px;
+  line-height: 1.5;
+  margin-top: 16px;
+  width: 25%;
+}
+
+.SecondaryHero-message-headline {
+  font-size: 24px;
+  line-height: 1.333;
+}
+
+.SecondaryHero-message-description {
+  padding: $padding-page 0;
+}
+
+.SecondaryHero-modules {
+  display: flex;
+  justify-content: space-between;
+  width: 70%;
+}
+
+.SecondaryHero-module {
+  align-items: center;
+  background-color: $white;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 0;
+  font-size: 18px;
+  justify-content: space-around;
+  line-height: 1.222;
+  margin: 16px;
+  padding: 0 16px 16px 16px;
+  position: relative;
+  text-align: center;
+  width: 33%;
+}
+
+.SecondaryHero-module-icon {
+  height: 60px;
+  margin: 16px;
```

Comment:
Did you make adjustments to the spacing in the mocks? The spacing looks a little different, like the space between the icon and text and the overall padding.

mock:

<img width="362" alt="Screenshot 2019-09-10 16 14 58" src="https://user-images.githubusercontent.com/55398/64651262-ae329480-d3e6-11e9-8eca-9fd7c0e16cd8.png">


site:

<img width="343" alt="Screenshot 2019-09-10 16 14 43" src="https://user-images.githubusercontent.com/55398/64651264-b1c61b80-d3e6-11e9-9c62-178985e6f313.png">

Reply:
Honestly I was just trying to match the mock (and when I say mock I mean the Sketch file) as closely as possible using css. I can see that the icons don't look the same as in the mocks, but I'm using the size from the mock (60px) so I'm not sure what else I can do to match it.

I've gone back to Sketch to look at all the spacing between objects and have tried to match it more exactly. I was hoping to make a "best guess" and then ask @brassy- for feedback on any changes that were needed, but I have made some improvements from v1 of this patch.